source/crud/crud.rst

```diff
@@ -1491,6 +1500,8 @@ Below are defined the exceptions that should be thrown from the various write me
 
   }
 
+See the `Bulk Write Specification </source/driver-bulk-update.rst#on-errors>`_ for more information on constructing errors from bulk writes.
```

Comment:
This spec is currently deprecated, should we be adding new links to it? There wasn't any replacement written for it once it was deprecated, so may be we have to.

Reply:
Hmm, I don't feel strongly either way. Perhaps we should wait until we have a new bulk spec. I removed it for now.